pyproject.toml

```diff
@@ -40,42 +190,23 @@ template = "changelog.d/_template.rst"
 # issue_format = "`#{issue} <https://github.com/python-attrs/attrs/issues/{issue}>`_"
 # issue_format = ":gh:`{issue}`"
 
-  # [[tool.towncrier.type]]
```

Comment:
The parts are already done by PR #375. The Towncrier parts below will possibly create a conflict when you rebase.

Reply:
yup, I shall exclude the #375 changes from this PR. Until now I was only doing PRs for my one-man show projects, never interfering with anyone else's changes upstream, So I picked up these nasty habits of mixing in other unrelated changes into the PR. I'll correct this. Happy to work with you on a more complex project and learn to produce some more disciplined PRs.